### ISSUE TITLE
404s not handled correctly

### ISSUE THREAD

**Issue by tomelm** [striglia/pyramid_swagger#178]
It looks like 404s aren't being handled correct:

```
ipdb> e
<RequestValidationError at 0x2b2f5c2190a8 400 Bad Request>
ipdb> e.code
400
ipdb> e.message
'Could not find a matching route for GET request http://telmalem.dev:9200/v3/foo. Have you registed this endpoint with Pyramid?'
```

If I hit a path that doesn't exist, it raises a validation error, says it can't find the route, and then raises a 400

**Comment by georgiev-martin** [comment]
That error message has a typo, too.

**Comment by striglia** [comment]
Curious...I'm surprised we don't have this covered in the existing testing suite. Should be easy to extend the existing acceptance testing to cover this case, then we can dig into why the code is a generic 400 not a 404.

**Comment by sjaensch** [comment]
Looks like the not yet documented `validate_path` setting could help you out: setting it to False should return a 404 for paths that are not in the spec as long as pyramid is not configured to handle them. #183 will help otherwise.

**Comment by sjaensch** [comment]
This was fixed by #191.
